agents/hybrid.py

```diff
@@ -177,3 +177,14 @@ def softmax(self, q_values, beta, p, top_stage_action=False, previous_action=Non
         probabilities = exp_values / exp_values.sum(axis=0)
 
         return probabilities
+
+    def reset(self):
```

Comment:
when is this called? why do we need it?

Reply:
I was staring at the void when I wrote it, it's not used anyway.